For example, pow() from mathlib is used directly in src/ffmpeg2theora.c:

<snip>
v = pow(v, g) * 255.0;    // mplayer's vf_eq2.c multiplies with 256 here, strange...
</snip>

For build failure and log, see Gentoo bug #504698

--- SConstruct
+++ SConstruct
@@ -206,9 +206,8 @@
   if env['crossmingw']:
       env.Append(CCFLAGS=['-Wl,-subsystem,windows'])
       env.Append(LIBS=['m'])
-  elif env['static']:
-      env.Append(LIBS=['m', 'dl'])
 
+  env.Append(LIBS=['m', 'dl'])
 
 env = conf.Finish()
 
